http-api/src/main/java/net/runelite/http/api/RuneLiteAPI.java

```diff
@@ -102,6 +102,13 @@ public static HttpUrl getApiRoot()
 
 	public static HttpUrl getApiBase()
 	{
+		final String prop = System.getProperty("runelite.api.url");
+
+		if (prop != null && !prop.isEmpty())
```

Comment:
https://google.github.io/guava/releases/19.0/api/docs/com/google/common/base/Strings.html#isNullOrEmpty(java.lang.String)

Reply:
I am aware of that, we don't use Guava in http-api